### ISSUE TITLE
support for sets

### ISSUE THREAD

**Issue by ponty** [taichino/prettyprint#2]
Sets are not supported by prettyprint (json error).
Solution: replace sets recursively with lists.

```
def replace_set(obj):
    if isinstance(obj, dict):
        for k,v in obj.items():
            if isinstance(v, set):
                obj[k]=list(v)
            replace_set(v)
    return obj
pp(replace_set(obj))
```

**Comment by taichino** [comment]
thank you very much for your comment!
and I'm very sorry for my so slow response...

I've added set support in version 0.1.4.
(but not sets.Set. this is deprecated module.)

**Comment by ponty** [comment]
It doesn't work if set is not top level.
This unit test fails:

```
def test_recursive():
    print pp_str([set(['x'])])
```

But I don't see an easy solution, so this problem may be ignored.

**Comment by taichino** [comment]
Thank you for your pull request!
I merged it in version 0.1.5.
